### PR TITLE
Add EIP: Reserve Precompile Address Range for RIPs

### PR DESCRIPTION
**Pull request by CarlBeek**
As discussed in ACDE 177, reserve precompile range `0x100` to `0x1ff` for RIPs.

### CODE REVIEW

File: EIPS/eip-75xx.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+eip: 75xx
```
Comment (lightclient) [inline EIPS/eip-75xx.md:2]: 7587

Comment (abcoathup) [inline EIPS/eip-75xx.md:2]:
```suggestion
eip: 7587
```
Assigned by lightclients

Comment (CarlBeek) [reply EIPS/eip-75xx.md]: Thanks @lightclient!

Comment (CarlBeek) [reply EIPS/eip-75xx.md]: Thanks @abcoathup!

File: EIPS/eip-7587.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+author: Carl Beekhuizen (@carlbeek), Ansgar Dietrichs (@adietrichs), Danny Ryan (@djrtwo), Tim Beiko (@timbeiko)
+discussions-to: https://ethereum-magicians.org/t/eip-75xx-reserve-precompile-address-range-for-rips-l2s/17828
+status: Draft
+type: Informational
+created: 2023-12-21
+---
+
+## Abstract
+
+This EIP reserves precompile ranges to ensure there are no conflicts with those used by the RIP process.
```
Comment (timbeiko) [inline EIPS/eip-7587.md:14]:
```suggestion
This EIP reserves precompile ranges to ensure there are no conflicts with those used by the Rollup Improvement Proposal (RIP) process.
```

Worth spelling out fully in the first usage.

**Review by timbeiko — APPROVED** [review]

**Review by gcolvin — APPROVED** [review]

**Pull request merged**